stdlib/public/Concurrency/Task.swift

```diff
@@ -76,6 +76,45 @@ import Swift
 /// like the reason for cancellation.
 /// This reflects the fact that a task can be canceled for many reasons,
 /// and additional reasons can accrue during the cancellation process.
+///
+/// ### Task closure lifetime
+/// Tasks initialized with a closure, representing the code that will be executed by this task.
+///
+/// Once this code has ran to completion, and the task therefore has completed, resulting in either
+/// a failure or result value, this closure is eagerly released.
+///
+/// Keeping the `Task` object retained, will not indefinitely keep the closure retained.
+/// This means that tasks rarely need to capture values as `weak`, since when they complete
+/// any references they hold are released (as the closure is released).
+///
+/// For example, in this example, it is not necessary to capture the actor as weak,
```

Comment:
Should probably replace the second “example” with something like “in the following snippet of code” or something because two uses of “example” are so close together here.

Reply:
Gotcha will do